### ISSUE TITLE
Output of some processes isn't logged

### ISSUE THREAD

**Issue by marcbachmann** [romabysen/exitd#2]
In my setup I'm launching 4 processes. Some don't log until I quit the process.
Is there some kind of buffering in place which could cause the issue?
Here's my repo if you want to debug it: https://github.com/upfrontIO/dockerfile-varnish

**Comment by romabysen** [comment]
Yeah, I think it's because the process buffers the output. I'm not really sure how to solve it though. Perhaps the stdout/stderr of the child process can be forced into non-buffering mode by exitd.

**Comment by romabysen** [comment]
Alternatively, at least with Python you can force your process not to buffer stdout/stderr by setting the environment variable PYTHONUNBUFFERED.
